integration-test/src/main/java/com/sequenceiq/it/cloudbreak/cloud/v4/mock/MockCloudProvider.java

```diff
@@ -436,4 +437,14 @@ private MockNetworkV1Parameters distroXNetworkParameters() {
         params.setVpcId(getVpcId());
         return params;
     }
+
+    @Override
+    public EnvironmentTestDto withResourceEncryption(EnvironmentTestDto environmentTestDto) {
+        return environmentTestDto;
+    }
+
+    @Override
+    public DistroXTestDtoBase withResourceEncryption(DistroXTestDtoBase distroXTestDtoBase) {
+        return distroXTestDtoBase;
+    }
```

Comment:
can environment and distrox encryption differ? right now you are returning with distrox dto in case of every provider, so my question is that why do we need a different method for distrox?

Reply:
it seems the distrox type of `withResourceEncryption` is not sued

Reply:
`withResourceEncryption` has been prepared for @satyabollineni to [CB-15358 Add e2e tests for cases having separate CMK for Datahub](https://jira.cloudera.com/browse/CB-15358)

Reply:
ohh ok, makes sense